docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -456,6 +456,42 @@ SELECT *
 FROM example.testdb.customer_orders FOR VERSION AS OF 3
 ```
 
+A different approach of retrieving historical data is to specify a point in time
+in the past, such as a day or week ago. The latest snapshot of the table taken
+before or at the specified timestamp in the query is internally used for
+providing the previous state of the table:
+
+```sql
+SELECT *
+FROM example.testdb.customer_orders FOR TIMESTAMP AS OF TIMESTAMP '2022-03-23 09:59:29.803 Europe/Vienna';
+```
+
+The connector allows to create a new snapshot through DeltaLake's [replace table](delta-lake-create-or-replace).
+
+```sql
+CREATE OR REPLACE TABLE example.testdb.customer_orders AS
+SELECT *
+FROM example.testdb.customer_orders FOR TIMESTAMP AS OF TIMESTAMP '2022-03-23 09:59:29.803 Europe/Vienna';
+```
+
+You can use a date to specify a point a time in the past for using a snapshot of a table in a query.
+Assuming that the session time zone is `Europe/Vienna` the following queries are equivalent:
+
+```sql
+SELECT *
+FROM example.testdb.customer_orders FOR TIMESTAMP AS OF DATE '2022-03-23';
```

Comment:
could you pls add coverage for this use case?

Reply:
It requires the commits in different date, do you have idea how to create it?

Reply:
Added the cases that using Date, but the cases would fail with `No temporal version history at or before` or ` Pointer value xxx is not in the past`.
Added a TODO to reminder if we could add a success case that using Date to time travel